source/modules.tex

```diff
@@ -827,12 +827,11 @@
 export module A;
 export inline void fn_e();      // error: exported inline function \tcode{fn_e} not defined
                                 // before private module fragment
-inline void fn_m();             // OK, module-linkage inline function
+inline void fn_m();             // error: module-linkage inline function \tcode{fn_m} not defined
```

Comment:
Maybe:
```suggestion
inline void fn_m();             // error: non-exported inline function \tcode{fn_m} not defined
                                // before private module fragment
```
The current comment makes it sound like the linkage matters here, and it doesn't; making this parallel the prior comment might make it clearer that exportedness doesn't matter either?

Reply:
Agreed, fixed.

Reply:
The fix has not been pushed.

Reply:
Now it has. Thanks for the hint.